dispatch/api/views.py

```diff
@@ -71,13 +71,17 @@ def get_queryset(self):
 
         q = self.request.query_params.get('q', None)
         section = self.request.query_params.get('section', None)
+        author = self.request.query_params.get('author', None)
 
         if q is not None:
             queryset = queryset.filter(headline__icontains=q)
 
         if section is not None:
             queryset = queryset.filter(section_id=section)
 
+        if author is not None:
+            queryset = queryset.filter(authors__id=author)
```

Comment:
Have you tested this when an article has multiple authors? Might be worth adding a test case.

Reply:
Didn't test it in the test case but when testing it out on the front end filtering by author=authorA catches articles with authors=[authorA, authorB]. I'll add a case.